examples/3dtiles.html

```diff
@@ -27,7 +27,10 @@
 
             var menuGlobe = new GuiTools('menuDiv', view, 300);
 
-            itowns.Fetcher.json('layers/JSONLayers/Ortho.json').then(function (result) { return view.addLayer(result) });
+            itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(function _(config) {
+                var layer = new itowns.ColorLayer(config.id, config);
```

Comment:
alternative 1

```js 
var addLayerGui = menuGlobe.addLayerGUI.bind(menuGlobe);

itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(function _(config) {
   var layer = new itowns.ColorLayer(config.id, config);
   view.addLayer(layer).then(addLayerGui);
});
``` 
alternative 2

```js 
itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(function _(config) {
   view.addLayer(new itowns.ColorLayer(config.id, config)).then(addLayerGui);
});
``` 
alternative 3
```js 
function addLayerColorFromConfig(config) {
   view.addLayer(new itowns.ColorLayer(config.id, config)).then(addLayerGui);
}

itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(addLayerColorFromConfig);
``` 

I prefer Alternative 3

Reply:
There is only one `ColorLayer` to add, I don't see the point of doing that ?

Reply:
it's a remark about the whole PR

Reply:
Ok, I rewrite all examples that have multiple layers of the same type added from a configuration file.